### ISSUE TITLE
Circular dependencies because of skinsdb and bakedclay/nether

### ISSUE THREAD

**Issue by antevens** [minetest-mods/3d_armor#89]
There are some dependency cycles involving these mods and I think 3d_armor might be the easiest place to fix these, if not I'm happy to file the issue with another mod.

```
3d_armor -> backedclay -> technic_cnc > technic -> i3 -> 3d_armor -> bakedclay
         -> nether -> riesenpilz -> technic -> i3 -> 3d_armor
                                                  -> skinsdb -> 3d_armor
                                                             -> clothing -> appliances -> technic
```

**Comment by BuckarooBanzay** [comment]
If you are using the `ethereal` mod, could this be the root cause: https://github.com/mt-mods/moretrees/issues/22 ?

**Comment by fluxionary** [comment]
IMO the bug here is that i3 shouldn't depend on 3d_armor nor on skinsdb.

additionally, 3d_armor shouldn't depend on bakedclay (it doesn't even seem to use it? that's an easy fix)

**Comment by wsor4035** [comment]
> IMO the bug here is that i3 shouldn't depend on 3d_armor nor on skinsdb.
> -snip-

i mean, it could not and use a register on mods loaded to handle that part, but at its core, it does provide features in itself that use apis/functionality from those mods, it should (optionally) depend on them

**Comment by fluxionary** [comment]
should this have really been closed? the bakedclay dependency was only part of the issue, and i don't think everyone has agreed w/ me that i3's dependencies are the problem and not 3d_armor's dependencies.

**Comment by wsor4035** [comment]
...its an autoclosed issue since you linked it in the pr https://github.com/minetest-mods/3d_armor/pull/90  
![image](https://user-images.githubusercontent.com/24964441/206959312-267cc18f-1aca-4122-8390-8583b1098e4e.png)

**Comment by fluxionary** [comment]
> ...its an autoclosed

i assumed so, but the autoclose is a natural language processing failure. a partial fix shouldn't have closed this. probably i could have phrased that differently? does github have a guide to how to not accidentally trigger the autoclose?

**Comment by tenplus1** [comment]
https://github.com/minetest-mods/3d_armor/pull/100

**Comment by BuckarooBanzay** [comment]
should be fixed with #100
